src/install-choose/install-choose-dialog.ts

```diff
@@ -267,7 +267,7 @@ class ESPHomeInstallChooseDialog extends LitElement {
           <mwc-circular-progress
             active
             ?indeterminate=${progress === undefined}
-            .progress=${progress !== undefined ? progress / 100 : undefined}
+            .progress=${progress !== undefined ? progress / 100 : 1}
```

Comment:
This should be using the `ifDefined` helper from lit.

Reply:
I've tried adding ifDefined, but I get this error then:
![image](https://github.com/esphome/dashboard/assets/1741838/2c0a39ac-00f4-4a6f-aba0-793590ee23b6)

not sure if this is a false-positive, but lit-plugin shows me many errors, including `Type 'true' is not assignable to 'string'` for empty scrimClickAction, ref: https://github.com/esphome/dashboard/pull/303